src/sentry/grouping/strategies/configurations.py

```diff
@@ -157,9 +157,20 @@ def register_strategy_config(id: str, **kwargs) -> type[StrategyConfiguration]:
     },
 )
 
+# This is simply a base for the hierarchical grouping until we can remove it
+register_strategy_config(
+    id="newstyle_hierarchical:1970-01-01",
```

Comment:
We create a base configuration so we can inherit from it and can then call `newstyle_hierarchical.py`.